examples/timer/Makefile

```diff
@@ -44,13 +44,13 @@ $(BUILD_DIR):
 	mkdir -p $(BUILD_DIR)
 
 $(BUILD_DIR)/client.o:
-	$(CC) -c $(CFLAGS) client.c -o $(BUILD_DIR)/client.o
+	$(CC) -c $(CFLAGS) -DTIMER_CHANNEL=1 client.c -o $(BUILD_DIR)/client.o
```

Comment:
I think for the example we should have it in the `client.c` file instead of the Makefile to be consistent with the serial and echo server examples.